packages/perseus-editor/src/widgets/__tests__/interactive-graph-editor.test.tsx

```diff
@@ -0,0 +1,445 @@
+import {ApiOptions, Dependencies} from "@khanacademy/perseus";
+import {render, screen} from "@testing-library/react";
+import {userEvent as userEventLib} from "@testing-library/user-event";
+import * as React from "react";
+
+import {testDependencies} from "../../../../../testing/test-dependencies";
+import InteractiveGraphEditor from "../interactive-graph-editor";
+
+const baseProps = {
```

Comment:
Would it be useful to type this as `Partial<PropsFor<typeof InteractiveGraphEditor>>`?  Or is it safe just because we pass it to the component in this file anyways so type inference should keep things correct?

Reply:
I think it's safe, but I'll add types anyway for clarity.